cli/Cargo.toml

```diff
@@ -61,6 +61,7 @@ walkdir = "2"
 xdg = "2.4"
 httpmock = "0.7"
 regex = "1.5"
+regress = "0.9.1"
```

Comment:
Wasn't sure which version to add, the progenitor README says to use 0.4 but it looks like they use 0.9.1 https://github.com/oxidecomputer/progenitor/blob/b3f59bfb9ab779df75180eea6e4071913a89f033/progenitor/Cargo.toml#L24